### DISCUSSION TITLE
Filter which layers export to shapefiles

### DISCUSSION THREAD

**Discussion by aliya-ryan** [digidem/mapeo-desktop#546] [Ideas]
Further to exporting map as shape files add the ability to filter data that is exported (e.g. same filters as display filters)

**Comment by okdistribute** [comment]
see mapeo/core here to implement this https://github.com/digidem/mapeo-core/blob/master/lib/export-shapefile.js

**Comment by okdistribute** [comment]
@aliya-ryan the shapefile export will now export into different files for each kind of layer, e.g., point line polygon. does that work for you?

**Comment by aliya-ryan** [comment]
@karissa yes this would be great if it did that - I haven't tested it yet though.
